runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -122,6 +130,10 @@ boolean findTag(int itemId, String search)
 
 		Collection<String> tags = getTags(itemId, false);
 		tags.addAll(getTags(itemId, true));
+		if (strict)
+		{
+			return tags.stream().anyMatch(tag -> tag.equals(Text.standardize(search)));
+		}
 		return tags.stream().anyMatch(tag -> tag.startsWith(Text.standardize(search)));
```

Comment:
This could probably be changed to be a bit more user friendly, no? 

```suggestion
		return tags.stream().anyMatch(tag -> tag.contains(Text.standardize(search)));
```